crates/libs/windows/src/imp/delay_load.rs

```diff
@@ -8,7 +8,7 @@ use super::*;
 ///
 /// * Both the library and function names must be valid null-terminated strings.
 pub unsafe fn delay_load<T>(library: core::PCSTR, function: core::PCSTR) -> Option<T> {
-    let library = LoadLibraryA(library);
+    let library = LoadLibraryExA(library, 0, LOAD_LIBRARY_SEARCH_DEFAULT_DIRS);
```

Comment:
Would it be better to use `LOAD_LIBRARY_SEARCH_SYSTEM32` here? Seems all internal use of delay_load only uses modules from \system32 context.

Reply:
No, both cppwinrt and the windows crate support xcopy activation for WinRT.

Reply:
@kennykerr